wgsl/index.bs

```diff
@@ -6371,13 +6371,13 @@ value with the same sign.
   <tr><td>`clamp(x)`<td>Correctly rounded
   <tr><td>`cos(x)`<td>Absolute error &le; 2<sup>-11</sup> inside the range of [-&pi;, &pi;]
   <tr><td>`cosh(x)`<td>Inherited from `(exp(x) - exp(-x)) * 0.5`
-  <tr><td>`cross(x, y)`<td>Inherited from `(x[i] * y[j] - x[j] * y[j])`
+  <tr><td>`cross(x, y)`<td>Inherited from `(x[i] * y[j] - x[j] * y[i])`
   <tr><td>`distance(x, y)`<td>Inherited from `length(x - y)`
   <tr><td>`exp(x)`<td>3 + 2 * |x| ULP
   <tr><td>`exp2(x)`<td>3 + 2 * |x| ULP
   <tr><td class="nowrap">`faceForward(x, y, z)`<td>Inherited from `select(-x, x, dot(z, y) < 0.0)`
   <tr><td>`floor(x)`<td>Correctly rounded
-  <tr><td>`fma(x, y, z)`<td>Inherited from `x * y + z`
+  <tr><td>`fma(x, y, z)`<td>1 ULP
```

Comment:
I have sad news.

The "inherited from  x *  y + z" looseness originates in GLSL (4.7.1 Range and precision), which flows through to GLSLStd450FMa in Vulkan "Precision of GLSL.std.450 instructions".

Need @alan-baker to review.

Reply:
I should say I don't really understand the analysis at https://github.com/gpuweb/gpuweb/pull/2109#issuecomment-919745407 yet